.github/workflows/integration_test.yml

```diff
@@ -26,7 +26,10 @@ jobs:
         run: |
           export CLUSTER="chart-testing"
           hack/local-up-chaos-mesh.sh
-
+          kubectl set env deployment/chaos-dashboard SECURITY_MODE=true -n chaos-testing
+          sleep 5
+          kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333 &
+      
```

Comment:
Should the integration test influence the `release-1.0` branch? This workflow will run when the `v1` branch changed (v1 has no support of auth).

Reply:
you are right, I removed the `release-1.0`